data_dir/large_collections.yaml

```diff
@@ -19,6 +19,9 @@ table_definition: |
     nums_set set<int>,
     names_set set<text>,
     books_set set<text>,
+    static_data1 blob static,
+    static_data2 blob static,
```

Comment:
nit-pick: Two columns are added, but the subject is 'a static column'

Reply:
using a static column requires using a clustering key.